pkg/frontend/adminactions/delete_managedresource.go

```diff
@@ -0,0 +1,72 @@
+package adminactions
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/Azure/go-autorest/autorest/azure"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/util/azureclient"
+	"github.com/Azure/ARO-RP/pkg/util/loadbalancer"
+)
+
+var (
+	frontendIPConfigurationPattern = `(?i)^/subscriptions/(.+)/resourceGroups/(.+)/providers/Microsoft\.Network/loadBalancers/(.+)/frontendIPConfigurations/([^/]+)$`
+	denyList                       = []string{
+		`(?i)^/subscriptions/(.+)/resourceGroups/(.+)/providers/Microsoft\.Network/privateLinkServices/([^/]+)$`,
```

Comment:
We want both PLS and PE (/providers/Microsoft.Network/privateEndpoints/)

Reply:
From my initial search the Private Endpoint resource does not exist in the cluster managed resource group, just the private link service (which references the PE)

Please double check.

Reply:
Added Private Endpoints to the deny list.

Reply:
PE would be the PE for the gateway, PLS would be our entrypoint into the cluster.  

Both should be blocked from deletion.